boards/riscv/beagle_v_starlight/beagle_v_starlight.yaml

```diff
@@ -0,0 +1,9 @@
+identifier: beagle_v_starlight
```

Comment:
as per @pdp7's recommendation, I would change to `beaglev_starlight_jh7100`

Reply:
Fixed.